### PR TITLE
Update events-admin.js for Bug #45612

### PR DESCRIPTION
**Pull request by jentheo**
Reset default view option if the selected view is disabled

### CODE REVIEW

File: src/resources/js/events-admin.js [hunk 1]
```diff
@@ -735,40 +735,54 @@ jQuery( document ).ready( function( $ ) {
...
 
-		var $default_view_select = $( '.tribe-field-dropdown_select2 select[name="viewOption"]' ),
-			$view_inputs         = $tribe_views.find( 'input:checkbox' ),
-			$view_desc           = $( '#tribe-field-tribeEnableViews .tribe-field-wrap p.description' ),
-			view_options         = {};
+		var $default_view_select = $( 'select[name="viewOption"]' ),
+			$default_mobile_view_select = $( 'select[name="mobile_default_view"]' ),
+			$view_inputs = $tribe_views.find( 'input:checkbox' ),
+			$view_desc = $( '#tribe-field-tribeEnableViews .tribe-field-wrap p.description' ),
+			view_options = {};
```
Comment (bordoni) [inline src/resources/js/events-admin.js]: move towards having multiple `var` statements.

File: src/resources/js/events-admin.js [hunk 2]
```diff
@@ -777,26 +791,37 @@ jQuery( document ).ready( function( $ ) {
...
+				$prev_default_mobile_option .attr( 'selected', 'selected' );
+			} else {
+				var $default_reset = $tribe_views.find( 'checkbox:checked' ).first().val();
+				$default_mobile_view_select.find( 'option' ).find( "option[value='" + $default_reset + "']" ).attr( 'selected', 'selected' );
 			}
 
 			$default_view_select
 				.select2( 'destroy' )
-				.select2( {width: '250px'} );
+				.select2( {width: 'auto'} );
```
Comment (bordoni) [inline src/resources/js/events-admin.js]: spacing around `{}`

**Comment by barryhughes** [comment]
Compiled the JS as requested :+1:

**Comment by barryhughes** [comment]
Now targeting `release/M17.20` (4.6.2) 

... Since we're starting smoketest for 17.19 it's a little too late to merge in there, but we can definitely get this in place in the next MR.

**Comment by barryhughes** [comment]
:cactus: 

@jentheo - looks good! Let's merge and get to QA

**Comment by barryhughes** [comment]
:ticket: [#45612](https://central.tri.be/issues/45612)

**Pull request merged**